src/DataProtection/README.md

```diff
@@ -36,7 +36,7 @@ This file contains the configuration for generating My API from the OpenAPI spec
 # it's the same options as command line options, just drop the double-dash!
 require:
   - $(this-folder)/../readme.azure.noprofile.md
-input-file: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/dataprotection/resource-manager/Microsoft.DataProtection/stable/2022-04-01/dataprotection.json
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/c0d5296b483a5fe4de7851fcd45acde14e736574/specification/dataprotection/resource-manager/Microsoft.DataProtection/stable/2022-05-01/dataprotection.json
```

Comment:
Please use `branch` such as this: https://github.com/Azure/azure-powershell/blob/a501c63bc3daddbf5695c706a3043a537fea1bca/src/Databricks/README.md?plain=1#L51

Reply:
sure, will do